docs/source/en/tasks/audio_classification.mdx

```diff
@@ -14,27 +14,44 @@ specific language governing permissions and limitations under the License.
 
 <Youtube id="KWwzcmG98Ds"/>
 
-Audio classification assigns a label or class to audio data. It is similar to text classification, except an audio input is continuous and must be discretized, whereas text can be split into tokens. Some practical applications of audio classification include identifying intent, speakers, and even animal species by their sounds.
+Audio classification assigns a label or class to audio data. It is similar to text classification, except an audio input is continuous and must be discretized, whereas text can be split into tokens. Some practical applications of audio classification include identifying speaker intent, language classification, and even animal species by their sounds.
```

Comment:
Actually think that the continuous / discrete statement is a bit confusing? IMO we want to drive the similarity between text and audio classification:
- Text input -> class label output
- Audio input -> class label output

=> same task, different input domain

Reply:
Good point! 👍